### PR TITLE
WF-62851-Get the palette color collection 

### PR DESCRIPTION
**Pull request by YuvarajPalanisamy**
Now included the section for getting the color collection for palette in the chart color model.

Regarding https://support.syncfusion.com/agent/tickets/370058 customer want to publish the ug document for getting the color of chart color palette. which is used for each customer to get color programmatically.

### CODE REVIEW

File: WindowsForms/chart/Chart-Appearance.md [hunk 1]
```diff
@@ -311,6 +311,25 @@ You can set the individual color for each segment of the series by using the Int
 
 ![Predefined palettes in Winforms Chart series](Chart-Appearance_images/Series_Palette.PNG)
 
+### Getting chart palette colors
+
+The GetColor() function of ColorModel can be used to acquire a list of colors for the palette in the chart control. The code example below demonstrates how to obtain the chart palette color list.
```
Comment (laksmipriyamurugesan) [inline WindowsForms/chart/Chart-Appearance.md:316]: The following code example demonstrates how to obtain the chart palette color list.

Comment (YuvarajPalanisamy) [reply WindowsForms/chart/Chart-Appearance.md]: Content review changes has been updated

File: WindowsForms/chart/Chart-Appearance.md [hunk 2]
```diff
@@ -311,6 +311,25 @@ You can set the individual color for each segment of the series by using the Int
 
 ![Predefined palettes in Winforms Chart series](Chart-Appearance_images/Series_Palette.PNG)
 
+### Getting chart palette colors
+
+The GetColor() function of ColorModel can be used to acquire a list of colors for the palette in the chart control. The following code example demonstrates how to obtain the chart palette color list..
+
```
Comment (MuneeshKumarG) [inline WindowsForms/chart/Chart-Appearance.md:317]: Remove the extra dot.

File: WindowsForms/chart/Chart-Appearance.md [hunk 3]
```diff
@@ -311,6 +311,25 @@ You can set the individual color for each segment of the series by using the Int
...
+    
+    int numberOfPaletteColor = this.chartControl1.Palette == ChartColorPalette.Metro ? ChartColorModel.NumColorsInMetroPalette : ChartColorModel.NumColorsInPalette;
+    
+    List<Color> paletteColors = new List<Color>();
+    for (int i = 0; i < numberOfPaletteColor; i++)
+    {
+        paletteColors.Add(this.chartControl1.Model.ColorModel.GetColor(i));
+    }
+	
+{% endhighlight %}
```
Comment (MuneeshKumarG) [inline WindowsForms/chart/Chart-Appearance.md:333]: Attach image

File: WindowsForms/chart/Chart-Appearance.md [hunk 4]
```diff
@@ -311,6 +311,25 @@ You can set the individual color for each segment of the series by using the Int
...
+
+The GetColor() function of ColorModel can be used to acquire a list of colors for the palette in the chart control. The following code example demonstrates how to obtain the chart palette color list..
+
+{% highlight c# %}
+
+    this.chartControl1 = new ChartControl();
+
+    . . .
+    
+    int numberOfPaletteColor = this.chartControl1.Palette == ChartColorPalette.Metro ? ChartColorModel.NumColorsInMetroPalette : ChartColorModel.NumColorsInPalette;
```
Comment (MuneeshKumarG) [inline WindowsForms/chart/Chart-Appearance.md:324]: numberOfPaletteColor -> numberOfPaletteColors

**Review by SyncfusionKarthikeyan — APPROVED** [review]
This is fine

**Pull request merged**